src/layoutcomponent/HamburgerNavbar.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const HamburgerNavbar = ({ showHamburgerMenu, setShowHamburgerMenu }) => {
+  return (
+    <div className={"hamburger_navbar " + (showHamburgerMenu ? "show" : "")}>
```

Comment:
You can use template literals instead of string concatenation

Reply:
okey i will use this henceforth